### ISSUE TITLE
new update button is missing color contrast for accessibility

### ISSUE THREAD

**Issue by tassoman** [piwik/piwik#20870]
When you have new matomo updates a button informs you about it.
Text is orange on gray, contrast `2.37` is not enough for WCAG 2.1 requirement 1.4.3

## Context

![immagine](https://github.com/matomo-org/matomo/assets/92727/4476e8ba-9d39-4a3a-9f59-cf55abc5129b)

## Expected Behavior

Color contrast must be valid for accessibility 

## Current Behavior

Color contrast is not enough

## Possible Solution
Remove color `#f57c00` and use regular black.

## Steps to Reproduce (for Bugs)

1. install an outdated version
2. inspect button pictured before

## Your Environment

* Matomo Version: 4.x
* PHP Version: 8.x
* Browser: any

**Comment by michalkleiner** [comment]
Hi @tassoman, thank you for the issue report.
We want the new update option standing out visually though we appreciate the a11y issue here. The design team will look at what options we might have to improve the contrast.
